target/bin/generate-dkim-config

```diff
@@ -4,29 +4,36 @@ touch /tmp/vhost.tmp
 
 # if no keysize is provided, 2048 is default.
 KEYSIZE=${1:-2048}
+# optional domain names
+DOMAINS=${2:=}
```

Comment:
Actually the tests are failing with:

`line 8: $2: cannot assign in this way`

Reply:
Why did you change it to default variable assignment (`:=`) from the original default variable usage (`:-`)?
The result shouldn't be a different as we are _assigning_ the value anyways to `DOMAINS`

Reply:
Take a look at [Bash Reference Manual - 3.5.3 Shell Parameter Expansion](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion)
```
${parameter:-word}
If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.

${parameter:=word}
If parameter is unset or null, the expansion of word is assigned to parameter. The value of parameter is then substituted. Positional parameters and special parameters may not be assigned to in this way.
```

Important is this one: **Positional parameters and special parameters may not be assigned to in this way.**

Reply:
I will revert the change back to `:-` during test implementation

Reply:
I wrote so many of them in `start-mailserver.sh` that I misinterpreted it here. In `start-mailserver.sh` assignments like this happen:

``` BASH
SOME_VAR="${SOME_VAR:=}"
```

which works obviously since it's the same variable. In this case, you're right, `:=` won't work since I was trying to give `$2` a new value. I was thinking about whether `:=` or `:-` but took the wrong one. Of course, `:-` is right. Thanks. As you said you'd revert this, I think this is resolved:)